scripts/setup_integration/kafka_integration_env.sh

```diff
@@ -26,7 +26,7 @@ start_podman () {
   podman run -d --pod=vector-test-integration-kafka --name vector_zookeeper $ZOOKEEPER
    # pinned to 2.6.0 due to https://github.com/wurstmeister/kafka-docker/issues/669
   podman run -d --pod=vector-test-integration-kafka -e KAFKA_BROKER_ID=1 \
-	 -e KAFKA_ZOOKEEPER_CONNECT=vector_zookeeper:2181 -e KAFKA_LISTENERS=PLAINTEXT://:9091,SSL://:9092,SASL_PLAINTEXT://:9093 \
+	 -e KAFKA_ZOOKEEPER_CONNECT=localhost:2181 -e KAFKA_LISTENERS=PLAINTEXT://:9091,SSL://:9092,SASL_PLAINTEXT://:9093 \
```

Comment:
Did you mean to keep this diff? This will also change with #10817

Reply:
Yes, this was required to get podman running the Kafka image. Yes, it will change with #10817, but I think that PR will have other conflicts with this one in the test code as well.